app/src/ui/lib/list/list.tsx

```diff
@@ -564,9 +564,9 @@ export class List extends React.Component<IListProps, IListState> {
 
     return (
       <Grid
-        aria-label={null!}
+        aria-label={''}
```

Comment:
This change isn't due to declaration updates as I wrote in the commit, but you'll see a runtime error instead in dev mode.